mw-webapp/src/logic/waysTable/columns.tsx

```diff
@@ -25,6 +26,14 @@ export const columns = [
      */
     cell: ({row}) => renderCellValue(row.original.uuid),
   }),
+  columnHelper.accessor<"uuid", string>("uuid", {
+    header: "Link to Way",
+
+    /**
+     * Cell with link to way page
+     */
+    cell: (uuid) => renderLinkInCell(`/way/${uuid.getValue()}`, "Way Page"),
```

Comment:
it is better to use separate service for navigation inside the app, it is hard to maintain template strings.

Reply:
I have no idea how to implement this

Reply:
ok skipt it fopr now